dependencies/pom.xml

```diff
@@ -140,6 +141,11 @@
                 <artifactId>jaeger-client</artifactId>
                 <version>${version.lib.jaegertracing}</version>
             </dependency>
+            <dependency>
```

Comment:
Can you add a comment to state we are managing this only to force an upgrade of the version used by jaeger-client